kythe/typescript/indexer.ts

```diff
@@ -844,21 +898,28 @@ class Visitor {
 
     const remoteSym = this.typeChecker.getAliasedSymbol(localSym);
     // This imported symbol can refer to a type, a value, or both.
-    const kImportValue = remoteSym.flags & ts.SymbolFlags.Value ?
-        this.getSymbolName(remoteSym, TSNamespace.VALUE) :
-        null;
-    const kImportType = remoteSym.flags & ts.SymbolFlags.Type ?
-        this.getSymbolName(remoteSym, TSNamespace.TYPE) :
-        null;
-    // Mark the local symbol with the remote symbol's VName so that all
+    // Attempt to create the Kythe VName for both the symbol's type and value
+    // and mark the local symbol with the remote symbol's VName so that all
     // references resolve to the remote symbol.
-    this.symbolNames.set(localSym, [kImportType, kImportValue, kImportType]);
+    let kType, kValue;
```

Comment:
Why kVarName if it's not a constant?

Reply:
Could you elaborate on your question?

Reply:
This is more of a style question: why did you name the variable `kType, kValue` instead of `type, value`? I thought the `k` prefix was specified for constants.

Reply:
`k` is prefixed on variables that are VNames, seemingly standing for `kythe`.

Reply:
Your guess is right.  This naming convention of mine is dumb, I am sorry.  I'd be happy to replace it with something that makes more sense.